common/azure-storage-common.gemspec

```diff
@@ -42,16 +42,9 @@ Gem::Specification.new do |s|
   s.required_ruby_version = ">= 2.3.0"
 
   s.add_runtime_dependency('faraday',                 '~> 1.0')
-  s.add_runtime_dependency('faraday_middleware',      '~> 1.0.0.rc1')
+  s.add_runtime_dependency('faraday_middleware',      "~> 1.0", ">= 1.0.0.rc1")
```

Comment:
One minor tweak: perhaps we could change this to just `"~> 1.0"` and encourage users to stop using the release candidate. Not super important if it would cause an issue for anyone.

Reply:
If this gem works with any 1.x version that would indeed seem to be enough. We (with help of Github) will make sure these dependencies are updated (periodically when an updated version is released).

This PR allows us to update Nokogiri and get rid of this security warning: https://github.com/advisories/GHSA-2rr5-8q37-2w7h (even though it is not applicable to CRuby, we don't want any open alerts ;))

Reply:
It would be best to make a minimum change to avoid any potential issues.